Sources/Plasma/NucleusLib/pnGameMgr/pnGmClimbingWall.h

```diff
@@ -0,0 +1,191 @@
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+Additional permissions under GNU GPL version 3 section 7
+
+If you modify this Program, or any covered work, by linking or
+combining it with any of RAD Game Tools Bink SDK, Autodesk 3ds Max SDK,
+NVIDIA PhysX SDK, Microsoft DirectX SDK, OpenSSL library, Independent
+JPEG Group JPEG library, Microsoft Windows Media SDK, or Apple QuickTime SDK
+(or a modified version of those libraries),
+containing parts covered by the terms of the Bink SDK EULA, 3ds Max EULA,
+PhysX SDK EULA, DirectX SDK EULA, OpenSSL and SSLeay licenses, IJG
+JPEG Library README, Windows Media SDK EULA, or QuickTime SDK EULA, the
+licensors of this Program grant you additional
+permission to convey the resulting work. Corresponding Source for a
+non-source form of such a combination shall include the source code for
+the parts of OpenSSL and IJG JPEG Library used as well as that of the covered
+work.
+
+You can contact Cyan Worlds, Inc. by email legal@cyan.com
+ or by snail mail at:
+      Cyan Worlds, Inc.
+      14617 N Newport Hwy
+      Mead, WA   99021
+
+*==LICENSE==*/
+
+#ifndef _pnGmClimbingWall_h_
+#define _pnGmClimbingWall_h_
+
+#include "pnGameMgr.h"
+#include "pnGmClimbingWallConst.h"
+
+/*****************************************************************************
+ *
+ *   Climbing Wall
+ *
+ ***/
+
+//============================================================================
+//	Game type id
+//============================================================================
+
+const plUUID kGameTypeId_ClimbingWall("6224cdf4-3556-4740-b7cd-d637562d07be");
+
+//============================================================================
+//	Network message ids
+//============================================================================
+
+// Cli2Srv message ids
+enum
+{
+    kCli2Srv_ClimbingWall_ChangeNumBlockers = kCli2Srv_NumGameMsgIds,
+    kCli2Srv_ClimbingWall_Ready,
+    kCli2Srv_ClimbingWall_BlockerChanged,
+    kCli2Srv_ClimbingWall_Reset,
+    kCli2Srv_ClimbingWall_PlayerEntered,
+    kCli2Srv_ClimbingWall_FinishedGame,
+    kCli2Srv_ClimbingWall_Panic,
+};
+
+// Srv2Cli message ids
+enum
+{
+    kSrv2Cli_ClimbingWall_NumBlockersChanged = kSrv2Cli_NumGameMsgIds,
+    kSrv2Cli_ClimbingWall_Ready,
+    kSrv2Cli_ClimbingWall_BlockersChanged,
+    kSrv2Cli_ClimbingWall_PlayerEntered,
+    kSrv2Cli_ClimbingWall_SuitMachineLocked,
+    kSrv2Cli_ClimbingWall_GameOver,
+};
+
+//============================================================================
+// Begin networked data structures
+#include <PshPack1.h>
```

Comment:
This isn't cross-platform, right?

Reply:
Probably not. It looks like the other files are doing

```C++
//============================================================================
// Begin networked data scructures
#pragma pack(push, 1)
//============================================================================
```